vignettes/SetSpecies.Rmd

```diff
@@ -116,18 +133,17 @@ simparams$species
 simparams$species_proportions
 ```
 
+### Single exophilic mosquito species 
 
-## IRS with single exophilic mosquito species 
+We will run the same model with IRS as above, but this time with an example mosquito species similar to *An. funestus*, but with a lower propensity to bite indoors (which we will name "exophilic"). Note that now there are 3 columns for the `ls_theta`, `ls_gamma`, `ks_theta`, `ks_gamma`, `ms_theta`, and `ms_gamma` arguments. See the Vector Control: IRS vignette for more information about setting IRS with different mosquito species. 
```

Comment:
I think there are only 2 columns for the 2 rounds of IRS?

Reply:
@kellymccain28 I think this was yours - can you confirm? It looks like two rows in the matrix to me (one for each timepoint).

Reply:
Yes nrows=length(timesteps), ncols=length(species)